geonode/harvesting/tasks.py

```diff
@@ -113,6 +113,10 @@ def _harvest_resource(
     worker: BaseHarvesterWorker = harvestable_resource.harvester.get_harvester_worker()
     harvested_resource_info = worker.get_resource(
         harvestable_resource, harvesting_session_id)
+    if worker.should_copy_resource(harvestable_resource):
+        copied_name = worker.copy_resource(harvestable_resource, harvested_resource_info)
```

Comment:
@afabiani this `copied_name` is put into the list in `HarvestedResourceInfo.copied_resources`. It is later passed over to the resource_manager.